### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by xavibou** [ragavsachdeva/CYWS-3D#4]
Hi,
Very interesting work! Do you plan on releasing the datasets and training code?

**Comment by ragavsachdeva** [comment]
Hi, sorry this has been on the back burner for too long. I'll get to it shortly. A lot of the datasets used are available [here](https://github.com/ragavsachdeva/The-Change-You-Want-to-See). I'll add the new ones. I'm not sure if I still have the training script, but the model has the loss function implemented (you can use a similar training script as [here](https://github.com/ragavsachdeva/The-Change-You-Want-to-See/blob/main/main.py)).

**Comment by xavibou** [comment]
Thanks for your quick reply! I understand that the training datasets are the same in both papers, and that the difference between the original and the (Now in 3D) one is the introduction of the DFRM module to warp and align features. Is that correct?
Best!

**Comment by ragavsachdeva** [comment]
Almost. KC-3D (generating using Kubric) and RC-3D datasets are new. For training this model, I used KC-3D + Coco-inpainted.

**Comment by xavibou** [comment]
Got it! Thanks again for your input :)

**Comment by TenderPower** [comment]
Hello author, where can I download the KC-3D datasets?

**Comment by ragavsachdeva** [comment]
@xavibou @TenderPower I've added the [datasets](https://github.com/ragavsachdeva/CYWS-3D?tab=readme-ov-file#datasets) and sample dataloaders. Feel free to let me know if you encounter any issues.

**Comment by TenderPower** [comment]
Hi,  where can I find the training code?
